web/core/components/core/description-versions/modal.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback, useRef } from "react";
+import { observer } from "mobx-react";
+import { ChevronLeft, ChevronRight, Copy } from "lucide-react";
+// plane imports
+import { EditorReadOnlyRefApi } from "@plane/editor";
+import { useTranslation } from "@plane/i18n";
+import { TDescriptionVersion } from "@plane/types";
+import { Avatar, Button, getButtonStyling, Loader, ModalCore, setToast, TOAST_TYPE, Tooltip } from "@plane/ui";
+import { calculateTimeAgo, cn, copyTextToClipboard } from "@plane/utils";
+// components
+import { RichTextReadOnlyEditor } from "@/components/editor";
+// hooks
+import { useMember, useWorkspace } from "@/hooks/store";
+
+type Props = {
+  activeVersionDescription: string | undefined;
+  activeVersionDetails: TDescriptionVersion | undefined;
+  handleClose: () => void;
+  handleNavigation: (direction: "prev" | "next") => void;
+  handleRestore: (descriptionHTML: string) => void;
+  isNextDisabled: boolean;
+  isOpen: boolean;
+  isPrevDisabled: boolean;
+  isRestoreDisabled: boolean;
+  projectId: string | undefined;
+  workspaceSlug: string;
+};
+
+export const DescriptionVersionsModal: React.FC<Props> = observer((props) => {
+  const {
+    activeVersionDescription,
+    activeVersionDetails,
+    handleClose,
+    handleNavigation,
+    handleRestore,
+    isNextDisabled,
+    isPrevDisabled,
+    isOpen,
+    isRestoreDisabled,
+    projectId,
+    workspaceSlug,
+  } = props;
+  // refs
+  const editorRef = useRef<EditorReadOnlyRefApi>(null);
+  // store hooks
+  const { getUserDetails } = useMember();
+  const { getWorkspaceBySlug } = useWorkspace();
+  // derived values
+  const activeVersionId = activeVersionDetails?.id;
+  const workspaceId = getWorkspaceBySlug(workspaceSlug?.toString() ?? "")?.id ?? "";
```

Comment:
Can you make sure `workspaceSlug` is defined before passing it to this method instead of passing an empty string? Also handle the workspaceId in the similar way?